### ISSUE TITLE
SARIF Report structure Validation Failing

### ISSUE THREAD

**Issue by sanyam803** [google-github-actions/analyze-code-security-scc#35]
### TL;DR

Current SARIF report is not compliant with the SARIF format. Current report has empty values for some of the key parameters.
SARIF report structure which includes not just the keys, but values should comply with SARIF structure. 
This tool [SARIF_Validator](https://sarifweb.azurewebsites.net/Validation) can help in validating the same. 
 

### Expected behavior

informationUri should be non empty. 
In general SARIF report should be validated against [SARIF_Validator](https://sarifweb.azurewebsites.net/Validation)

### Observed behavior

"informationUri": ""
informationUri is empty and the generated report is non compliant with [SARIF_Validator](https://sarifweb.azurewebsites.net/Validation)

### Action YAML

```yaml
N/A
```

**Comment by sanyam803** [comment]
The above bug was closed in this commit : https://github.com/google-github-actions/analyze-code-security-scc/pull/36
